neqo-http3/src/server.rs

```diff
@@ -446,8 +457,8 @@ mod tests {
     }
 
     // Connect transport, send and receive settings.
-    fn connect() -> (Http3Server, PeerConnection) {
-        let (mut hconn, mut neqo_trans_conn) = connect_and_receive_settings();
+    fn create_client_and_connect_with_server(server: &mut Http3Server) -> PeerConnection {
```

Comment:
A shorter name might be easier.

```suggestion
    fn connect_to(server: &mut Http3Server) -> PeerConnection {
```